build.gradle.kts

```diff
@@ -79,6 +79,9 @@ dependencies {
 
 	implementation("com.fasterxml.jackson.module:jackson-module-parameter-names:$jacksonModuleVersion")
 
+    //temporal
+	implementation("io.temporal:temporal-spring-boot-starter:$temporalVersion")
```

Comment:
@fvuotoeng should we depend on the spring-boot-starter or just to `io.temporal.temporal-sdk` ?

Reply:
On samples available at https://github.com/temporalio/samples-java/tree/main/springboot-basic and https://github.com/temporalio/samples-java/tree/main/springboot we can see both implementation "org.springframework.boot:spring-boot-starter-web"
and implementation "org.springframework.boot:spring-boot-starter-web".